### PR TITLE
[MIG][9.0] medical prescription thread 

### PR DESCRIPTION
**Pull request by woodbrettm**
Hi there,

Migration of medical_prescription_thread to v9.

Cheers,
Brett

### CODE REVIEW

**Review by lasley — APPROVED** [review]
Thanks @bmw95 

Let's change the name of this module to fit OCA guidelines better though. `mail_thread_medical_prescription`

@t3ddftw please review

**Comment by lasley** [comment]
Note Travis is failing due to unrelated issue now that our testing is turned on in Travis file (#138). @obulkin please see LABS-358 & apply the abstract model testing strategy you came up with into a new PR please.

**Review by susport — COMMENTED** [review]
The code is ok and I test in runbut is 👍 . But the travis error has to do the test.

**Comment by obulkin** [comment]
The testing issue should be resolved now: https://github.com/OCA/vertical-medical/pull/142

**Comment by woodbrettm** [comment]
Thanks @obulkin :). Will rebase later tonight.

**Comment by lasley** [comment]
I'll tackle it in a bit actually, some of my branches need rebasing too so it'll be easy to do them all at once

**Comment by woodbrettm** [comment]
oh sick thanks @lasley.

**Comment by lasley** [comment]
@bmw95 - still need to change name to mail_thread_medical_prescription

File: medical_prescription_thread/README.rst [hunk 1]
```diff
@@ -1,46 +1,45 @@
...
-Odoo Medical Prescription Threading
-===================================
+===========================
+Medical Prescription Thread
+===========================
 
 Extension of medical_prescription that adds message threading functionality
-    to Prescription Orders and Prescription Order Lines
-
+to prescription orders and prescription order lines.
```
Comment (lasley) [inline medical_prescription_thread/README.rst:10]: Note - this ReadMe looks fine regarding #147. Simple module, simple ReadMe

**Comment by woodbrettm** [comment]
@lasley good to go

**Review by lasley — APPROVED** [review]
Thanks @bmw95

**Pull request merged**